src/main/java/io/nats/client/support/NatsJetStreamConstants.java

```diff
@@ -68,6 +68,9 @@ public interface NatsJetStreamConstants {
     // JSAPI_DIRECT_GET is the endpoint to directly get a message.
     String JSAPI_DIRECT_GET = "DIRECT.GET.%s";
 
+    // JSAPI_DIRECT_GET is the shortcut endpoint to directly get a last by subject message
```

Comment:
better comment - is shortcut the right word

Reply:
done